plugin/src/test/java/com/epam/grandhackathon/deployment/sphere/plugin/metadata/persistence/dao/DeploymentMetaDataDaoMock.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.grandhackathon.deployment.sphere.plugin.metadata.persistence.dao;
+
+import java.util.List;
+
+import com.epam.grandhackathon.deployment.sphere.plugin.metadata.model.DeploymentMetaData;
+
+public class DeploymentMetaDataDaoMock extends DeploymentMetaDataDao {
```

Comment:
Since I can't partially mock DeployVersionMetaDataCollector because its final, in order to test it I need real instance. Creating one requires DeploymentMetaDataDao and EnvironmentDao injected, so I created mocks for these dao objects same way it was done in BuildVersionMetaDataCollectorTest before.